src/main/protobuf/v1/catalog-item.proto

```diff
@@ -47,8 +47,11 @@ message CatalogDescriptorV1 {
   required int32 dailyCallsTotal = 10;
   optional AgreementApprovalPolicyV1 agreementApprovalPolicy = 11;
   optional int64 createdAt = 12;
-  optional int64 activatedAt = 13;
-  repeated string serverUrls = 14;
+  repeated string serverUrls = 13;
```

Comment:
protobuf use a positional fields definition.
This means that for existing records, the field at position 13 is an `int64`, and in our case has the meaning of activatedAt/publishedAt.

In order to keep retrocompatibility with existing data, the field at position 13 must be assigned to the `publishedAt` field